### DISCUSSION TITLE
[1093910] Tales of the Black Forest [本体]

### DISCUSSION THREAD

**Discussion by XiaoMing-111** [pjy612/SteamManifestCache#372] [Share]
appid:
`appid:1093910`
keys:
```
1093911;815e0377ea5af4f16ae800ccbcb5337a6e818bbbc92d8fe904b2634e2d3f5229
```
Code:47C4B0799A1F89D7
Email:xmxmxm10101@outlook.com
[1093911_7741418719007693618.zip](https://github.com/pjy612/SteamManifestCache/files/13987094/1093911_7741418719007693618.zip)

**Comment by pjy612** [comment]
帖子格式正确。
有对应的清单么？ 有的话 麻烦您编辑原帖 补上。

**Reply by XiaoMing-111** [reply]
补上啦

**Reply by XiaoMing-111** [reply]
还差什么吗

**Reply by pjy612** [reply]
不用 看邮箱

**Reply by XiaoMing-111** [reply]
还没有收到，垃圾邮件也看了

**Reply by XiaoMing-111** [reply]
有了有了
